docs/sources/developer/Developers-Guide.md

```diff
@@ -50,51 +36,74 @@ Overview of the general architecture of Plaso:
 
 ## Roadmap
 
-A high level roadmap can be found [here](../user/Releases-and-roadmap.md). Individual features are tracked as a github issue and labeled as "enhancement". A list of features can be found [here](https://github.com/log2timeline/plaso/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement).
+A high level roadmap can be found [here](../user/Releases-and-roadmap.md). 
+Individual features are tracked as a github issue and labeled as "enhancement".
+A list of features we'd already like to add can be found 
+[here](https://github.com/log2timeline/plaso/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement).
 
 ## Contributing Code
 
-Want to add a parser to Plaso and you are ready to go? Start by checking [here](https://github.com/log2timeline/plaso/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement) if someone is already working on it. If you don't see anything there you can just go ahead and [create an issue on the github site](https://github.com/log2timeline/plaso/issues) and mark it as "enhancement". Assign the issue to yourself so that we can keep track on who is working on what.
+Want to add a parser to Plaso and you are ready to go? Start by checking 
+[here](https://github.com/log2timeline/plaso/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement) 
+if someone is already working on it. If you don't see anything there you can 
+just go ahead and [create an issue on the github site](https://github.com/log2timeline/plaso/issues) 
+and mark it as "enhancement". Assign the issue to yourself so that we can keep 
+track on who is working on what.
 
-If you cannot program and still have a great idea for a feature please go ahead and create an issue and leave it unassigned, note that the priority will be who ever wants to work on it. Or consider this the idea opportunity to learn yourself Python programming.
+If you cannot program and still have a great idea for a feature please go ahead 
+and create an issue and leave it unassigned, note that the priority will be who 
+ever wants to work on it.
 
-Before you start writing the code, please review the following:
+Before you start writing  code, please review the following:
 
-* [Style guide](Style-guide.md). All code submitted to the project needs to follow this style guide.
-* [Code review](Codereview.md). All code that is submitted into the project needs to be reviewed by at least one other person.
-* [Adding a new dependency](https://github.com/log2timeline/l2tdocs/blob/master/process/Dependencies.md). If your code requires adding a new dependency please check out these instructions.
+* [Style guide](Style-guide.md). All code submitted to the project needs to 
```

Comment:
Wasn't this moved to l2tdocs?

Reply:
There's a l2t style guide, but there are Plaso specific things as well - tests, and some more naming stuff to come.

Reply:
ack